classes/class.ilObjORCAContentGUI.php

```diff
@@ -294,6 +294,13 @@ protected function saveFormValues()
         $DIC->ctrl()->returnToParent($this);
     }
 
+    function endsWith($FullStr, $needle)
+    {
+        $StrLen = strlen($needle);
+        $FullStrEnd = substr($FullStr, strlen($FullStr) - $StrLen);
+        return $FullStrEnd == $needle;
```

Comment:
please use only kebab_case in variable names (`$full_str`, `$str_len`, `$full_str_end`).